types/stellar-sdk/index.d.ts

```diff
@@ -18,7 +18,7 @@ export class Account {
 
 export class CallBuilder<T extends Record> {
     constructor(serverUrl: string)
-    call(): Promise<CollectionPage<T>>;
+    call(): Promise<CollectionPage<T> | T>;
```

Comment:
Is this a breaking change for the other builders since you'll have to disambiguate which type you're expecting?

Would it make sense to have something like `PagedCallBuilder<T> extends CallBuilder<CollectionPage<T>>` and then make `call()` return `Promise<T>` -- and change the other types of builders to extend from that?

Reply:
Yes, totally true.

On it.